src/vst.cpp

```diff
@@ -302,7 +314,7 @@ tresult PLUGIN_API Regrader::process( ProcessData& data )
     IParameterChanges* outParamChanges = data.outputParameterChanges;
     // a new value of VuMeter will be sent to the host
     // (the host will send it back in sync to our controller for updating our editor)
-    if ( outParamChanges && outputGainOld != outputGain ) {
+    if ( !isDoublePrecision && outParamChanges && outputGainOld != outputGain ) {
```

Comment:
For some reason Reaper goes insane when we're returning the VVU. Just ignore it.